mock-core-configs/etc/mock/templates/centos-stream-10.tpl

```diff
@@ -34,7 +35,89 @@ user_agent={{ user_agent }}
 name=CentOS Stream $releasever - Koji Local - BUILDROOT ONLY!
 baseurl=https://kojihub.stream.centos.org/kojifiles/repos/c{{ releasever }}s-build/latest/$basearch/
 cost=2000
+enabled=0
+
+[baseos]
+name=CentOS Stream $releasever - BaseOS
+#baseurl=http://mirror.stream.centos.org/$releasever-stream/BaseOS/$basearch/os/
+metalink=https://mirrors.centos.org/metalink?repo=centos-baseos-$releasever-stream&arch=$basearch&protocol=https,http
```

Comment:
There's an experimental commit now using this Metalink URL, which is IMO supposed to work (eventually) but it isn't (yet).  At the same time, the modified baseurl above works -> so I assume there's some temporary mirror synchronization issue?

Reply:
That's why I suggested gradual improvements.

Reply:
But the `baseurl` I provided here works, so I assume it is a matter of hours till the mirrors get synced?

Reply:
I don't know. Maybe, maybe not.

Reply:
I think we can wait till tomorrow or something, or report a mirroring issue.

Is there a reason to use the compose URL instead of the mirror baseurl?

Reply:
That's what the c10s package is using.

Reply:
Looks like the metalinks are still being set up: https://gitlab.com/redhat/centos-stream/rpms/centos-release/-/merge_requests/54

Reply:
Seems to work now :partying_face: